recipe/build-r-base.sh

```diff
@@ -540,9 +540,10 @@ Darwin() {
     make install ${EXTRA_MAKE_ARGS}
 
     pushd ${PREFIX}/lib/R/etc
-      sed -i -r "s|-isysroot ${CONDA_BUILD_SYSROOT}||g" Makeconf
-      sed -i -r "s|$BUILD_PREFIX/lib/gcc|$PREFIX/lib/gcc|g" Makeconf
-      sed -i -r "s|$BUILD_PREFIX/lib/gcc|$PREFIX/lib/gcc|g" Makeconf-r
+      sed -i'.bak' -r "s|-isysroot ${CONDA_BUILD_SYSROOT}||g" Makeconf
+      sed -i'.bak' -r "s|$BUILD_PREFIX/lib/gcc|$PREFIX/lib/gcc|g" Makeconf
+      sed -i'.bak' -r "s|-lemutls_w|-lgcc_s.1.1|g" Makeconf
```

Comment:
Let's go with just
```suggestion
      sed -i'.bak' -r "s|-lemutls_w||g" Makeconf
```
If there are openmp programs they'll have to use `-fopenmp` which will bring in the appropriate `libgcc`.

Reply:
Okay; you certainly know better what to do here :).